### ISSUE TITLE
When starting Rails 6 server receive "already initialized constant API_VERIFY_URL" Warning

### ISSUE THREAD

**Issue by ishields** [davidesantangelo/emailhunter#10]
When booting a Rails 6 server I'm receiving a warning 
```
/lib/email_hunter/account.rb:4: warning: already initialized constant API_VERIFY_URL
/lib/email_hunter/verify.rb:4:  warning: previous definition of API_VERIFY_URL was here
```
Looking at both files, this constant does seem duplicated but has different values.  Should this constant be moved into the class it self?

**Comment by davidesantangelo** [comment]
update gem to 0.8.1

**Comment by ishields** [comment]
Hello! I updated the gem to 0.8.1 and it's still happening unfortunately.  
```
Fast Debugger (ruby-debug-ide 0.8.0.beta18, debase 0.3.0.beta18, file filtering is supported, block breakpoints supported, smart steps supported, obtaining return values supported, partial obtaining of instance variables supported) listens on 0.0.0.0:56080

.../.rvm/gems/ruby-2.6.3/gems/emailhunter-0.8.1/lib/email_hunter/account.rb:4: warning: already initialized constant API_VERIFY_URL

.../.rvm/gems/ruby-2.6.3/gems/emailhunter-0.8.1/lib/email_hunter/verify.rb:4: warning: previous definition of API_VERIFY_URL was here

=> Booting WEBrick
=> Rails 6.0.0 application starting in development http://0.0.0.0:3005
=> Run `rails server --help` for more startup options
```

**Comment by ishields** [comment]
@davidesantangelo Curious if you've been able to repro this?  Let me know if you need more details or help figuring it out.

**Comment by ishields** [comment]
Not sure what's wrong but I pointed my branch to master and the error went away.  Maybe the version on rubygems is an old version or something like that?

gem 'emailhunter', git: 'git@github.com:davidesantangelo/emailhunter.git', branch: 'master'

**Comment by davidesantangelo** [comment]
fixed now! update to 0.8.3.
